.github/workflows/psql-release-notifier.yaml

```diff
@@ -0,0 +1,57 @@
+# #############################################################################
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# #############################################################################
+
+name: PostgreSQL new release notifier
+
+on:
+    workflow_dispatch:
+    schedule:
+        # Run every day at 01:00 at night
+        - cron: "0 1 * * *" # UTC time
+env:
+    TRACTUSX_MAILINGLIST: _CHANGE_ME_@MAIL.COM 
+    DEVSECOPS_NOTIFICATION_EMAIL: eclipse.tractusx@gmail.com
+    DEVSECOPS_NOTIFICATION_EMAIL_PASSWORD: ${{ secrets.NOTIFICATION_EMAIL_PASSWORD }}
+    CURRENT_ALIGNED_PSQL_VER: 15.0.0
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup Go 
+      uses: actions/setup-go@v4
+      with:
+        go-version: '>=1.20.7'
+    - name: Download artifact
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: ${{ github.event.workflow_run.workflow_id }}
+        workflow_conclusion: success
+        if_no_artifact_found: warn
+        name: psql_release
+        path: release-notifier/
+    - name: Run notifier
+      run: |
+        cd release-notifier
+        go run . psql
```

Comment:
I would prefer to specifically run `main.go` here

Reply:
Corrected.